Modules/_struct.c

```diff
@@ -76,40 +76,6 @@ typedef struct {
 #define PyStructObject_CAST(op)     ((PyStructObject *)(op))
 #define PyStruct_Check(op, state)   PyObject_TypeCheck(op, (PyTypeObject *)(state)->PyStructType)
 
-/* Define various structs to figure out the alignments of types */
```

Comment:
To reduce the diff, how about just redefining the macros with `_Alignof`?

Reply:
Not sure this helps.  Those macros used only once.